### ISSUE TITLE
build_node_compilers: build.yaml unable to set dart2js_args

### ISSUE THREAD

**Issue by paulreimer** [pulyaevskiy/node-interop#36]
I used to set them like this in `build.yaml`:

```
targets:
  $default:
    sources:
      - "lib/**"
      - "node/**"
    builders:
      build_node_compilers|entrypoint:
        options:
          compiler: dart2js
          dart2js_args:
            - --minify
            - --fast-startup
```

But now I get this error when I run `pub run build_runner build --output=build/`:
```
[SEVERE] build_node_compilers|entrypoint:

Invalid argument (dart2js_args): Expected a list of strings, but got a YamlList:: Instance of 'YamlList'
[SEVERE] build_node_compilers|entrypoint:
Failed to instantiate builder for <name> with configuration:
{
 "dart2js_args": [
  "--minify",
  "--fast-startup"
 ],
 "compiler": "dart2js"
}
error Command failed with exit code 78.
```

**Comment by pulyaevskiy** [comment]
Thanks for reporting! Fix is on the way, see #37.

**Comment by pulyaevskiy** [comment]
Published as 0.1.8.

**Comment by paulreimer** [comment]
Thanks, 0.1.8 works for me!
